cmake/Program.cs

```diff
@@ -202,14 +205,14 @@ private static void serialize()
                     0,
                     0,
                     localTime3dCrs);
-                w1i1SeismicFrameRep.setMdValues(logMds.cast(), 5, hdfProxy);
+                w1i1SeismicFrameRep.setMdValues(logMds.cast(), 5);
                 DoubleArray logTimes = new DoubleArray(5);
                 logTimes.setitem(0, 0);
                 logTimes.setitem(1, 10);
                 logTimes.setitem(2, 20);
                 logTimes.setitem(3, 25);
                 logTimes.setitem(4, 30);
-                w1i1SeismicFrameRep.setTimeValues(logTimes.cast(), 5, hdfProxy);
+                w1i1SeismicFrameRep.setTimeValues(logTimes.cast(), 5);
```

Comment:
logTimes should be GC kept alive in order to prevent to be collected before setMdValues finishes.

Reply:
Done.